rollup.config.js

```diff
@@ -0,0 +1,63 @@
+import typescript from '@rollup/plugin-typescript';
+import json from '@rollup/plugin-json';
+import shebang from '@walrus/rollup-plugin-shebang';
+import multiInput from 'rollup-plugin-multi-input';
+import fs from "fs";
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import pkge from './package.json'
+
+let filepaths = [];
+
+const walk = (root) => {
+  let currentPaths = fs.readdirSync(root);
+  for (let path of currentPaths) {
+    const abspath = `${root}/${path}`;
+    const isFile = /.*\.(ts|js|json)$/.test(path);
+
+    if (isFile) {
+      filepaths.push(abspath);
+    } else {
+      // isDirectory
+      walk(abspath);
+    }
+  }
+
+}
+
+walk('./src/commands');
+filepaths.push("./src/index.ts");
+
+// exclude type and test files
+filepaths = filepaths
+  .filter(path => !path.includes("test"))
+  .filter(path => !path.includes("type"));
+
+// console.log(filepaths);
+
+// exclude all dependencies from being bundled together, except for the problematic jsonLd, used by "open-attestation", "oa-verify" dependencies
+let deps = [...Object.keys(pkge.dependencies), ...Object.keys(pkge.devDependencies)]
+deps = deps.filter( (dep) => !(/oa\-|open-attestation/.test(dep)) )
+
+// console.log(deps);
+
+export default {
+  input: filepaths,
+  output: {
+    dir: 'rollup-build/cjs',
+    format: 'cjs',
+    exports: 'named'
+
+  },
+  external: [...deps],
+  plugins: [
+      resolve(),
+      commonjs(),
+      typescript(),
+      json(),
+      shebang({
```

Comment:
why need this ?

Reply:
commonJS: Convert CommonJS modules to ES6, so they can be included in a Rollup bundle. Rollup only recognises es6 js files.
typescript(): Convert typescript to js so that rollup can process. rollup only works on js files by default
 json(): Rollup only recognises js file. json() s.json files to ES6 modules
shebang(): to preserver the shebang line `#!/usr/bin/env node` in `src/index.ts`

Reply:
will provide comments